frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantModal/VariantOverrides/VariantOverrides.tsx

```diff
@@ -0,0 +1,163 @@
+                                    legalValues && legalValues.length > 0
+                                )}
+                                show={
+                                    <Autocomplete
+                                        multiple
+                                        id={`override-select-${index}`}
+                                        isOptionEqualToValue={(
+                                            option,
+                                            value
+                                        ) => {
+                                            return option === value;
+                                        }}
+                                        options={legalValues}
+                                        onChange={updateSelectValues(index)}
+                                        getOptionLabel={option => option}
+                                        value={filteredValues}
+                                        style={{ width: '100%' }}
+                                        filterSelectedOptions
+                                        size="small"
+                                        renderInput={params => (
+                                            <StyledTextField
+                                                {...params}
+                                                variant="outlined"
+                                                label="Legal values"
+                                            />
+                                        )}
+                                    />
+                                }
+                                elseShow={
+                                    <StyledInputListField
+                                        label="Values (v1, v2, ...)"
+                                        name="values"
+                                        placeholder=""
+                                        values={override.values}
+                                        updateValues={updateValues(index)}
+                                    />
+                                }
+                            />
+                            <Tooltip title="Remove" arrow>
+                                <IconButton
+                                    onClick={event => {
+                                        event.preventDefault();
+                                        overridesDispatch({
+                                            type: 'REMOVE',
+                                            payload: index,
+                                        });
+                                    }}
+                                >
+                                    <Delete />
+                                </IconButton>
+                            </Tooltip>
+                        </StyledFieldColumn>
+                    </StyledRow>
+                );
+            })}
+        </>
+    );
+};
```

Comment:
Mostly an updated duplicate of `frontend/src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/AddFeatureVariant/OverrideConfig/OverrideConfig.tsx`.